android/src/main/java/com/doublesymmetry/trackplayer/service/MusicService.kt

```diff
@@ -438,9 +440,10 @@ class MusicService : HeadlessJsTaskService() {
         return MusicBinder()
     }
 
-    fun destroyIfAllowed(ignoreStopWithAppSetting: Boolean = false) {
+    // TODO: #AEX-45 forceDestroy is needed when calling destroy() manually. Find an alternative solution that does not require a second flag.
```

Comment:
Added a todo and a separate internal ticket for this, since I think there can be a better solution for this :)